### PR TITLE
Feature/729 Update documentation for Settlements

### PR DESCRIPTION
**Pull request by ggrg**
[Story #729](https://github.com/mojaloop/documentation/pull/30)

### CODE REVIEW

File: mojaloop-technical-overview/account-lookup-service/README.md [hunk 1]
```diff
@@ -15,65 +15,65 @@ Use-cases that have been implemented over and above for Hub Operational use:
   
 ## 1. Design Considerations
 
-### 1.1 Account Lookup Service (ALS)
+### 1.1. Account Lookup Service (ALS)
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/README.md]: Note that the standard used throughout most of the documentation repo does not follow this standard.

Specifically, the main headers would end with a `.` but sub-headings would not.

There are two options here:
1. undo these changes with regard to the `.` dots. 
2. Otherwise, the rest of the documentation repo should be aligned to the same format

Comment (ggrg) [reply mojaloop-technical-overview/account-lookup-service/README.md]: @mdebarros, after performing a [search](https://www.google.com/search?q=subheading+full+stop) on the topic I found opinions in both directions: [option 1](https://www.proof-reading-service.com/en/blog/full-stops-titles-headings-captions/) and [option 2](http://www.sussex.ac.uk/informatics/punctuation/essaysandletters/titlesandheadings). As we already have **option 2** in place for all plantuml diagrams and my preference is to have it (as I don't see a reason why it would be removed for levels > 1), so I changed all files as requested.
